grouper/fe/templates/audit-create.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+
+{% block heading %}
+    <a href="/audits">Audits</a>
+{% endblock %}
+
+{% block subheading %}
+    Start Global Audit
+{% endblock %}
+
+{% block content %}
+<div class="col-md-6 col-md-offset-3">
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h3 class="panel-title">Start Global Audit</h3>
+       </div>
+        <div class="panel-body">
+            <form class="form-horizontal" role="form"
+                  method="post" action="/audits/create">
+                    <p>Submitting this form will create a new global audit. This will force
```

Comment:
I like that this page explains what's going on.  I think it would be better to say that an audit comprises separate "reviews" rather than "audits" for each group.